Project.toml

```diff
@@ -48,7 +48,7 @@ ArrayInterfaceCore = "0.1.22"
 ArrayInterfaceGPUArrays = "0.1, 0.2"
 ArrayInterfaceStaticArrays = "0.1"
 ArrayInterfaceStaticArraysCore = "0.1.2"
-DataStructures = "0.18"
+DataStructures = "0.18, 0.19"
```

Comment:
was this tested? Odd this hasn't come up in a CompatHelper?

Reply:
I was trying to dev DataStructures. This should be reverted. I eventually just changed the deved DataStructures version back to 0.18 instead.